manifests/e/eza-community/eza/0.20.2/eza-community.eza.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: eza-community.eza
+PackageVersion: 0.20.2
+PackageLocale: en-US
+Publisher: eza community
+PublisherUrl: https://eza.rocks
+PublisherSupportUrl: https://github.com/eza-community/eza/issues
+PackageName: eza
+PackageUrl: https://github.com/eza-community/eza
+License: MIT
+LicenseUrl: https://github.com/eza-community/eza/blob/main/LICENCE

```

```suggestion
LicenseUrl: https://github.com/eza-community/eza/blob/main/LICENSE.txt
```
Backing automation needs to have this updated.